week1/react-exercises/src/components/Hobby.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import uuid from 'uuid';
+
+function Hobby({ title, icon }) {
+  return (
+    <li key={uuid()}>
```

Comment:
Still if you have a `Hobby` Component it will have the key not it's children as:
```jsx
[<Hobby key={uuid()} />]
```

And we don't need this here, but the Component Jsx Element does 👆🏻:
```suggestion
    <li>
```

Reply:
Wow, now I get it! thanks a lot :bouquet: